web/src/rendered_markdown.ts

```diff
@@ -243,16 +259,18 @@ export const update_elements = ($content) => {
         }
     });
 
-    $content.find("span.timestamp-error").each(function () {
-        const [, time_str] = /^Invalid time format: (.*)$/.exec($(this).text());
+    $content.find("span.timestamp-error").each(function (): void {
+        const regex = /^Invalid time format: (.*)$/.exec($(this).text());
```

Comment:
This is a match array, not a `RegExp` object.

Reply:
renamed, thanks!